WorkflowRxSwift/Tests/WorkerTests.swift

```diff
@@ -0,0 +1,215 @@
+/*
+ * Copyright 2020 Square Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+import RxSwift
+import WorkflowRxSwift
+import WorkflowRxSwiftTesting
+import WorkflowTesting
+import XCTest
+@testable import Workflow
```

Comment:
I’d avoid `@testable` importing a module other than the one under test (i.e. treat this module as a third-party module that can’t access internals of Workflow). Can we use `WorkflowTesting` to help us here?